sky/serve/server/server.py

```diff
@@ -110,3 +110,18 @@ async def tail_logs(
         logs_path=request_task.log_path,
         background_tasks=background_tasks,
     )
+
+
+@router.post('/download-logs')
```

Comment:
```suggestion
@router.post('/sync-down-logs')
```
lets use the same identifier with the function name?